pkg/utils/informer/informermanager.go

```diff
@@ -255,6 +205,50 @@ func (s *informerManagerImpl) Stop() {
 	s.cancel()
 }
 
+func (s *informerManagerImpl) AddEventHandlerToInformer(resource schema.GroupVersionResource, handler cache.ResourceEventHandler) {
+	s.resourcesLock.Lock()
+	defer s.resourcesLock.Unlock()
+
+	// Get or create the informer (this is idempotent - if it exists, we get the same instance)
+	// The idempotent behavior is important because this method may be called multiple times,
+	// potentially concurrently, and relies on the shared informer instance from the factory.
+	informer := s.informerFactory.ForResource(resource).Informer()
+
+	// Add the event handler to the informer
+	_, _ = informer.AddEventHandler(handler)
```

Comment:
what are the return?

Reply:
I am handling the 2nd return value, which is an error, and document what is the first return value